config/locales/en.yml

```diff
@@ -1228,6 +1236,7 @@ en:
   notice_unable_delete_time_entry: "Unable to delete time log entry."
   notice_unable_delete_version: "Unable to delete version."
   notice_user_missing_authentication_method: User has yet to chose a password or another way the sign in.
+  notice_value_information: "Your %{key_name} is: %{value}"
```

Comment:
Grammar comes to mind when I think of it. But in the end you are right, I will try to give some hints about the usage of this string in its new name